libs/shared/console-shared/src/lib/job-configure-settings/ui/job-configure-settings.tsx

```diff
@@ -42,47 +48,41 @@ export function JobConfigureSettings(props: JobConfigureSettingsProps) {
             }}
             render={({ field, fieldState: { error } }) => (
               <InputText
-                className="mb-2"
                 name={field.name}
                 onChange={field.onChange}
                 value={field.value}
                 label="Schedule - Cron expression"
+                hint={
+                  formatCronExpression(watchSchedule) ? formatCronExpression(watchSchedule) + ` (${watchTimezone})` : ''
+                }
                 error={error?.message}
               />
             )}
           />
-          <div className="mb-3 flex justify-between">
-            <p className="text-neutral-400 text-xs">
-              {formatCronExpression(watchSchedule) ? formatCronExpression(watchSchedule) + ` (${watchTimezone})` : ''}
-            </p>
-            <ExternalLink href="https://crontab.guru" size="xs">
-              CRON expression builder
-            </ExternalLink>
-          </div>
-          <TimezoneSetting className="mb-3" />
-          <EntrypointCmdInputs />
-        </>
+          <TimezoneSetting />
+          {/* XXX: This should be migrated to general settings page */}
+          {props.legacyMode && <EntrypointCmdInputs />}
```

Comment:
Can you create a ticket for this ? I'm afraid we'll forget it

Reply:
Done https://qovery.atlassian.net/browse/FRT-1073